app/views/witnesses/new.html.erb

```diff
@@ -162,6 +162,7 @@
           <label>
             <input type="checkbox" ng-model="witness.available_day4">
             <%= t 'shared.available_day4' %>
+            <%= t 'shared.holocaust_day' %>
```

Comment:
1. check with Gili on this
2. PR's should be opened to develop
3. please always link the trello card to the pr